jupyterlab_h5web/utils.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from typing import Union
+
+
+def _resolve_path(file_path: Path, base_dir: Path) -> Path:
+    if file_path.is_absolute():
+        return file_path
+
+    return base_dir / file_path
+
+
+def resolve_file_path(
+    file_path: Union[str, Path], base_dir: Union[str, Path]
+) -> Union[str, Path]:
+    path = file_path if isinstance(file_path, Path) else Path(file_path)
+    base = base_dir if isinstance(base_dir, Path) else Path(base_dir)
+
+    if isinstance(file_path, Path):
+        return str(_resolve_path(path, base))
+
+    return _resolve_path(path, base)
```

Comment:
Maybe simpler without using `_resolve_path`, something like:

```suggestion
    abs_path = path if path.is_absolute() else (base / path)
    return abs_path if isinstance(abs_path, Path) else str(abs_path)
```

Reply:
In the end, I will enforce the use of `Path`. Less `os` imports and easier to read.